src/snippets/reference/bun/ClientOverride.js

```diff
@@ -36,10 +36,6 @@ export default {
       if (result.reason.isRateLimit()) {
         console.log("Rate limit rule", result);
       }
```

Comment:
Pehaps you update these a little more so it can be
```suggestion
      if (result.reason.isRateLimit()) {
        console.log("Rate limit rule result", result);
      } else {
        console.log("Rule result", result);
      }
```
And also remove the console.log at the top of the loop. This would have the currently configured rule and any added rules if they copy-paste

Reply:
Yeah good idea, I've changed it in a couple other examples too so as not to duplicate the logs.